src/components/ProductMilestoneCloseResultDetailPage/ProductMilestoneCloseResultDetailPage.tsx

```diff
@@ -135,9 +136,15 @@ export const ProductMilestoneCloseResultDetailPage = () => {
               </TextContent>
             </ToolbarItem>
           </Toolbar>
-          <PushedBuildsList pushedBuilds={closeResult?.buildPushResults} />
+          <ContentBox marginBottom shadow={false} background={false}>
+            <PushedBuildsList pushedBuilds={closeResult?.buildPushResults} />
+          </ContentBox>
 
           {/* TODO: Log*/}
```

Comment:
is this TODO for different log than you just added?

Reply:
It's TODO for the actual live log. I only added links to the old UI, not logs.